deploy/docs/Best_Practices.md

```diff
@@ -1095,3 +1098,55 @@ The reliability of the system can be calculated using the following formulas:
 
 - If limited by queue_size: `number_of_instances*timeout[min]*sending_queue.queue_size/load_in_DPM` minutes.
 - If limited by PVC size: `number_of_instances*PVC_size/(1KB*load_in_DPM)` minutes.
+
+## Assigning Pod to particular Node
+
+### Using NodeSelectors
+
+Kubernetes offers a feature of assigning specific pod to node. Such kind of control is sometimes useful,
+whenever you want to ensure that pod will end up on specific node according your requirements like operating system
+or connected devices.
+
+### Binding pods to linux nodes
```

Comment:
```suggestion
#### Binding pods to linux nodes
```

This IMHO should be nested under the section above.

Reply:
ok